reducers/storage.ts

```diff
@@ -110,6 +111,47 @@ function storage(state: Record<string, any> = {}, action: GenericAction) {
     }
 }
 
+function migrateDrafts(state: any) {
+    const drafts: any = {};
```

Comment:
Could we give `state` and `drafts` a proper type here?

Reply:
I played around with it and it gets a bit weird, our state is not strictly the `PersistedState` type, and if we try to work around it gives errors as the migration will only handle `PersistedState`.  I found this https://github.com/rt2zz/redux-persist/issues/1065 as well which is what we're facing here. I'm not a fan of using `any` as well.